api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -230,11 +238,34 @@ private WorkspaceExportRow toWorkspaceExportRow(DbWorkspace workspace) {
   }
 
   private WorkspaceExportRow toWorkspaceExportRow(DbUser user) {
+    String verifiedInstitutionName =
+        verifiedInstitutionalAffiliationDao
+            .findFirstByUser(user)
+            .map(via -> via.getInstitution().getDisplayName())
+            .orElse("");
+
     WorkspaceExportRow row = new WorkspaceExportRow();
     row.setCreatorContactEmail(user.getContactEmail());
     row.setCreatorUsername(user.getUsername());
+    row.setInstitution(verifiedInstitutionName);
     row.setCreatorFirstSignIn(
         user.getFirstSignInTime() == null ? "" : dateFormat.format(user.getFirstSignInTime()));
```

Comment:
nit: I'd prefer the nullable/map/orElse pattern here but I'm not gonna block on that

Reply:
Agreed, done